modules/conflict_resolver/test/conflict_resolverTest.php

```diff
@@ -97,7 +97,8 @@ function testConflictResolverPermission()
         $this->checkPagePermissions(
             '/conflict_resolver/',
             [
-                'conflict_resolver'
+                'conflict_resolver',
```

Comment:
function testConflictResolverPermission()
    {
         $this->setupPermissions(["conflict_resolver","data_dict_edit","data_dict_view"]);
         $this->safeGet($this->url . "/conflict_resolver/");
        $bodyText = $this->safeFindElement(
            WebDriverBy::cssSelector("body")
        )->getText();
        $this->assertStringNotContainsString(
            "You do not have access to this page.",
            $bodyText
        );
        $this->assertStringNotContainsString(
            "An error occured while loading the page.",
            $bodyText
        );
        $this->resetPermissions();
}
please try to use this function to replace the old one.